src/Component.elm

```diff
@@ -1,9 +1,32 @@
-module Component exposing (materialIcon)
+module Component exposing (empty, materialIcon, progressBar)
 
 import Html as H
 import Html.Attributes as HA
+import RemoteData
+
+
+empty : H.Html msg
+empty =
+    H.div [] []
```

Comment:
Technically for this purpose `H.text ""` is more suitable, because it render exactly nothing. Afaik nothing will be added to the DOM as opposed to empty `div` in your case.